src/lpython/semantics/python_ast_to_asr.cpp

```diff
@@ -809,9 +832,33 @@ class BodyVisitor : public CommonVisitor<BodyVisitor> {
         v.n_body = body.size();
     }
 
+    ASR::symbol_t* overloaddef_find_helper(std::string func_name, Vec<ASR::ttype_t*> args,
+                const Location &loc) {
+        for(auto &t: overload_defs[func_name]) {
+            bool ok = ASRUtils::select_func_subrout(t, args, loc,
+            [&](const std::string &msg, const Location &loc) { throw SemanticError(msg, loc); });
```

Comment:
I'm trying to figure how to find the `overload_defs` in `BodyVisitor`, since it was defined in `SymbolTable` visitor.